SimG4Core/CustomPhysics/src/G4SQInelasticCrossSection.cc

```diff
@@ -0,0 +1,45 @@
+
+#include "G4SystemOfUnits.hh"
+#include "G4DynamicParticle.hh"
+#include "G4NistManager.hh"
+
+#include "SimG4Core/CustomPhysics/interface/G4SQ.h"
+#include "SimG4Core/CustomPhysics/interface/G4AntiSQ.h"
+#include "SimG4Core/CustomPhysics/interface/G4SQInelasticCrossSection.h"
+
+G4SQInelasticCrossSection::G4SQInelasticCrossSection(double mass) : G4VCrossSectionDataSet("SQ-neutron") {
+  nist = G4NistManager::Instance();
+  theSQ = G4SQ::SQ(mass);
+  theAntiSQ = G4AntiSQ::AntiSQ(mass);
+}
+
+G4SQInelasticCrossSection::~G4SQInelasticCrossSection() {}
+
+G4bool G4SQInelasticCrossSection::IsElementApplicable(const G4DynamicParticle* aPart, G4int Z, const G4Material*) {
+  return ((0 < Z) && (aPart->GetDefinition() == theSQ || aPart->GetDefinition() == theAntiSQ));

```

Comment:
here enough "return true" - this x-section will not be used in any other case.

Reply:
Ok, done.